ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputShadowNode.java

```diff
@@ -49,7 +52,7 @@
 
   public ReactTextInputShadowNode() {
     mTextBreakStrategy = (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) ?
-        0 : Layout.BREAK_STRATEGY_SIMPLE;
+      Layout.BREAK_STRATEGY_SIMPLE : Layout.BREAK_STRATEGY_HIGH_QUALITY;
```

Comment:
is this change of behavior correct?

Reply:
BREAK_STRATEGY_HIGH_QUALITY was introduced in 23 or M, and is a default text breaking strategy. Before that BREAK_STRATEGY_SIMPLE was default, which has value of 0.